src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -1660,6 +1690,12 @@ inline void rpp_store8_f32_to_f32_avx(Rpp32f *dstPtr, __m256 *p)
     _mm256_storeu_ps(dstPtr, p[0]);
 }
 
+inline void rpp_store8_f16_to_f16_avx(Rpp16f *dstPtr, __m256 *p)
```

Comment:
Where is this used?

Reply:
It is not used any where . I removed that .